### PR TITLE
update the slot spi demo

### PR DESCRIPTION
**Pull request by yunfeiyanggzq**
### Describe what this PR does / why we need it
update the slot spi demo

### Describe how to verify it
the output is as following
```
INFO: Sentinel log output type is: file
INFO: Sentinel log charset is: utf-8
INFO: Sentinel log base directory is: C:\Users\m1881\logs\csp\
INFO: Sentinel log name use pid is: false
Current context: sentinel_default_context
Current entry resource: abc
Exiting for entry on DemoSlot: abc

Process finished with exit code 0

```

### CODE REVIEW

**Comment by yunfeiyanggzq** [comment]
please review, @sczyh30.Thanks!

File: sentinel-demo/sentinel-demo-slot-chain-spi/src/main/java/com/alibaba/csp/sentinel/demo/slot/DemoSlotChainBuilder.java [hunk 1]
```diff
@@ -20,17 +20,11 @@
 import com.alibaba.csp.sentinel.slots.DefaultSlotChainBuilder;
 
 /**
- * An example slot chain builder. To activate this slot chain builder,
- * add the class name to corresponding SPI file in `resource/META-INF/services` directory.
- *
  * @author Eric Zhao
+ *
+ * @deprecated since 1.7.2, we can use @SpiOrder(-3500) to adjust the order of {@link DemoSlot},
+ * this class is reserved for compatibility with older versions.
```
Comment (cdfive) [inline sentinel-demo/sentinel-demo-slot-chain-spi/src/main/java/com/alibaba/csp/sentinel/demo/slot/DemoSlotChainBuilder.java:26]: If has no annotation, the order is `SpiOrder.LOWEST_PRECEDENCE` and the slot will be at the end of slot chain.
How about adding `@SpiOrder(-3500)` in `DemoSlot` class?  
As the comment mentioned, the `DemoSlot` will be after `SystemSlot` and before `FlowSlot`,  and the demo is more friendly for new users.

Comment (yunfeiyanggzq) [reply sentinel-demo/sentinel-demo-slot-chain-spi/src/main/java/com/alibaba/csp/sentinel/demo/slot/DemoSlotChainBuilder.java:26]: good idea,done

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**